pkg/version/generate/release_generate.go

```diff
@@ -0,0 +1,110 @@
+// +build release
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/blang/semver"
+	"github.com/dave/jennifer/jen"
+
+	"github.com/weaveworks/pctl/pkg/version"
+)
+
+const versionFilename = "pkg/version/release.go"
+const defaultPreReleaseID = "dev"
+const defaultReleaseCandidate = "rc.0"
+
+func main() {
+	if len(os.Args) < 2 {
+		log.Fatal("missing argument")
+	}
+
+	command := os.Args[1]
+
+	var newVersion, newPreRelease string
+	switch command {
+	case "release":
+		newVersion, newPreRelease = prepareRelease()
+	case "release-candidate":
```

Comment:
I think for this project we decided against doing the whole release candidate dance

Reply:
actually, which of these command cases do we need/use?

Reply:
not sure, removed the released candidate one. do you want `full-version|print-version|print-major-minor-version` tp be removed too?

Reply:
🤔  not really sure off the top of my head. do we know what the github action which i guess will end up calling this will use? is the Makefile using anything?

Reply:
ok so I have removed `print-version|print-major-minor-version` don't think we need it